README.md

```diff
@@ -30,6 +31,18 @@ Visit the full documentation for [recommended install options](https://www.openc
 - [Prometheus Metrics](https://www.opencost.io/docs/integrations/prometheus)
 - Reference [User Interface](https://github.com/opencost/opencost/tree/develop/ui)
 
+## Using the UI
+
+The OpenCost UI provides a visualization of the Kubernetes allocations and the related Cloud Costs.
+
```

Comment:
Is this the best place for this info? Is there not a Usage page/guide that we can add to?

Reply:
it's also in the [UI README](https://github.com/opencost/opencost/tree/develop/ui#readme) but not everyone's going to leave the base README

Reply:
@dwbrown2 merge?

Reply:
Can we add a "Usage" section to docs nav and move this later? If so, I'm good to merge.

Reply:
@mattray updates? planning to remove this?